app/views/contracted-hours.html

```diff
@@ -20,11 +20,15 @@ <h2 class="heading-large">Contracted hours</h2>
 
 <p class="lede">Change the contracted hours for <span class="bold">{{title}}</span></p>
 
+{% if errors['hours'][0] %}
+  <span class="error-message">{{ errors['hours'][0] }}</span>
+{% endif %}
+
 <form action="/offender-manager/{{woId}}/contracted-hours" method="post" class="sln-form-action">
   <input type="hidden" name="sub_page" value="true">
   <div class="form-group">
```

Comment:
The error is rendered as a link which has a href to the anchor 'hours'. In order for this to work correctly there should be a corresponding element with 'hours' as the id.